src/api/json/catalog.json

```diff
@@ -8321,6 +8321,18 @@
       "description": "Manifest YAML files for AWS Copilot services, environments, and pipelines. Documentation: https://aws.github.io/copilot-cli/docs/manifest/overview/",
       "fileMatch": ["**/copilot/**/manifest.yml"],
       "url": "https://sootyowl.github.io/aws-copilot-schemas/copilot-schema.json"
+    },
+    {
+      "name": "Nx Schema",
+      "description": "Configuration file for the Nx CLI and project defaults. Documentation: https://nx.dev/reference/nx-json",
+      "fileMatch": ["nx.json"],
+      "url": "https://raw.githubusercontent.com/nrwl/nx/master/packages/nx/schemas/nx-schema.json"
+    },
+    {
+      "name": "Nx Project Schema",
+      "description": "Configuration file for the Nx CLI and project defaults. Documentation: https://nx.dev/reference/project-configuration",
+      "fileMatch": ["project.json"],
```

Comment:
can you find a less generic name for the file match? Could you perhaps include the folder name that it always is located in, for instance?